docs/source/api_reference/regression.rst

```diff
@@ -63,6 +63,14 @@ take one or multiple ``sklearn`` estimators and adda probabilistic prediction mo
 
     ResidualDouble
 
+.. currentmodule:: skpro.regression.multiquantile
+
+.. autosummary::
```

Comment:
I think this is the source of the readthedocs problem, but I cannot figure out what would b the problem. All seems fine to me.